MediaBrowser.Controller/Channels/IChannel.cs

```diff
@@ -64,7 +64,7 @@ public interface IChannel
         /// </summary>
         /// <param name="type">The type.</param>
         /// <param name="cancellationToken">The cancellation token.</param>
-        /// <returns>Task{DynamicImageInfo}.</returns>
+        /// <returns>Task{DynamicImageResponse}.</returns>
```

Comment:
NOTE: this should contain a description, not the type.

Reply:
Should I change that here or can I just leave it?

Reply:
You can leave it for now